account-info/src/lib.rs

```diff
@@ -1,25 +1,22 @@
 extern crate alloc;
 
```

Comment:
Not related to this PR specifically, but since multiple modules were added, should this module be called ```lib.rs```? What is the convention?  ```contract.rs``` seems to be a much more descriptive name. What do you think?